daemon/oci_linux.go

```diff
@@ -480,9 +480,10 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
+			data := c.HostConfig.Tmpfs[m.Destination]
```

Comment:
there should probably be a way to have this into `volume.MountPoint` so that is already set at _create_ in https://github.com/docker/docker/pull/23301/files#diff-a961c4e0039634ade1cd360227f858d8R122